aziotctl/Cargo.toml

```diff
@@ -18,7 +18,7 @@ foreign-types-shared = "0.1"
 hyper = "0.14"
 hyper-openssl = "0.9"
 libc = "0.2"
-nix = "0.18"
+nix = "0.22"
```

Comment:
Why is this needed?

And you might as well update to 0.23 then (the current latest).